fuse/eval/metrics/libs/classification.py

```diff
@@ -138,14 +140,20 @@ def accuracy(
     ):
         """
         Compute accuracy score
-        :param pred: class prediction. Each element is an integer in range [0 - num_classes)
-        :param target: the target class. Each element is an integer in range [0 - num_classes)
+        :param pred: class prediction. Each element is an integer in range [0 - num_classes) or a float in range [0-1]
```

Comment:
It's not a float but a numpy array (dtype=float) - score per class, right?
Can you please also mention that in such a case argmax will be applied?